src/conversion.jl

```diff
@@ -26,7 +26,7 @@ function generatorproducer(m::RepMatrix)
             while true
                 for col in 0:getd(m)
                     output = getsolution(m, col)
-                    if output !== nothing
+                    if length(output) != 0
```

Comment:
Was this a bug ?

Reply:
No. Please see my reply to your comment on `getsolution`.